examples/spring-boot-tomcat/src/main/java/example/springframework/boot/tomcat/HelloConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package example.springframework.boot.tomcat;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+import org.apache.catalina.Service;
+import org.apache.catalina.connector.Connector;
+import org.springframework.boot.web.embedded.tomcat.TomcatWebServer;
+import org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.healthcheck.HealthChecker;
+import com.linecorp.armeria.server.tomcat.TomcatService;
+import com.linecorp.armeria.spring.ArmeriaServerConfigurator;
+
+/**
+ * Configures an Armeria {@link Server} to redirect the incoming requests to the Tomcat instance provided by
+ * Spring Boot. It also sets up health check endpoint so that it works well with a load balancer.
+ */
+@Configuration
+public class HelloConfiguration {
+
+    /**
+     * Extracts a Tomcat {@link Connector} from Spring webapp context.
+     */
+    public static Connector getConnector(ServletWebServerApplicationContext applicationContext) {
+        final TomcatWebServer container =
+                (TomcatWebServer) applicationContext.getWebServer();
+        try {
+            final Field serviceConnectorsField = TomcatWebServer.class.getDeclaredField("serviceConnectors");
```

Comment:
You can simplify this reflection using following code.
```java
        TomcatWebServer container =
                (TomcatWebServer) applicationContext.getWebServer();
        // Starts the container explicitly to make sure all connectors are write back from serviceConnectors.
        container.start();
        Connector tomcatConnector = container.getTomcat().getConnector();
```

Reply:
w00t! Thanks a lot. :+1: